client/allocrunner/taskrunner/artifact_hook_test.go

```diff
@@ -126,7 +126,7 @@ func TestTaskRunner_ArtifactHook_PartialDone(t *testing.T) {
 	require.NoError(t, ioutil.WriteFile(file2, []byte{'1'}, 0644))
 
 	// Mock TaskRunner by copying state from resp to req and reset resp.
-	req.PreviousState = helper.CopyMapStringString(resp.State)
+	req.PreviousState = helper.CopyMap(resp.State)
```

Comment:
While we're at it we might as well kill our CopyMap in favor of https://pkg.go.dev/golang.org/x/exp/maps#Clone